src/server/ocpp/utils/OCPPUtils.ts

```diff
@@ -822,7 +820,9 @@ export default class OCPPUtils {
       stopMeterValues.push({
         id: (id++).toString(),
         ...meterValueBasedProps,
-        value: (transaction.currentInstantAmps ? transaction.currentInstantAmps : transaction.currentInstantAmpsDC),
+        value: (transaction.currentInstantAmps
+          ? transaction.currentInstantAmps / Utils.getNumberOfConnectedPhases(await ChargingStationStorage.getChargingStation(tenantID, transaction.chargeBoxID), null, transaction.connectorId)
```

Comment:
We could add the charging station as a parameter to the method. So we can avoid this extra database call.
It should be available in both locations, where the method is called.

Reply:
Done.